src/mixer/basetrackplayer.h

```diff
@@ -41,6 +42,8 @@ class BaseTrackPlayer : public BasePlayer {
     void loadTrackFailed(TrackPointer pTrack);
     void newTrackLoaded(TrackPointer pLoadedTrack);
     void unloadingTrack(TrackPointer pAboutToBeUnloaded);
+    void noPassthroughInputConfigured();
+    void noVinylControlInputConfigured();
 };
 
 class BaseTrackPlayerImpl : public BaseTrackPlayer {
```

Comment:
Qt uses an Abstract prefix for interface classes. We should follow this. 
How about: 
"TrackPlayerAbstract" 
"TrackPlayerBase" 
This would be unambiguous to me.